packages/block-library/src/blocks/slideshow/edit.js

```diff
@@ -88,8 +74,6 @@ class Edit extends Component {
 				<SlideshowPreview
 					{ ...newProps }
 					previewImage={ galleryImages[ selectedIndex ] }
-					onPrevArrowClick={ this.onPrevArrowClick.bind( this ) }
-					onNextArrowClick={ this.onNextArrowClick.bind( this ) }
 				/>
 
 				<InspectorControls { ...{ ...newProps, setIndex, selectedIndex } } />
```

Comment:
Change Media button should not be displayed inside block controls when the source attribute is set to posts